.eslintrc.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
Can this eslint change be extracted into a separate PR?

Reply:
Sure but what do you have in mind here?
I was thinking of implementing multiple tests to have a baseline template, although there are still failing tests.